presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -338,6 +329,24 @@ private JdbcOutputTableHandle beginWriteTable(ConnectorTableMetadata tableMetada
         }
     }
 
+    private String getColumnString(boolean uppercase, ImmutableList.Builder<String> columnNames, ImmutableList.Builder<Type> columnTypes, ColumnMetadata column)
+    {
+        String columnName = column.getName();
+        if (uppercase) {
+            columnName = columnName.toUpperCase(ENGLISH);
+        }
+        columnNames.add(columnName);
```

Comment:
This should be outside the function?

Reply:
Because `columnNames`/`columnTypes` are stateful, they won't be null in the handle.  But logically it doesn't belong in this method so I'll extract it.